src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -79,11 +80,36 @@ g.test('zero_size')
     };
 
     const success = zeroArgument === 'none';
+
     t.expectValidationError(() => {
       t.device.createTexture(descriptor);
     }, !success);
   });
 
+g.test('dimension_type_on_formats')
+  .desc(
+    `Test every dimension types on every formats. Note that 1) compressed formats are not valid for 1D and 3D dimension types and, 2) depth/stencil formats are not valid for 3D dimension type.`
```

Comment:
Just to confirm, 1d depth/stencil textures are valid? I don't think you can render into 1d textures, so not sure what the usefulness is. Even if the APIs support it (why?) I wouldn't be surprised if it's an underutilized path that turns out to be buggy...

Reply:
Yeah. Just noted the comments that depth/stencil formats are invalid for 1D on Metal (from the log Austin put on dawn issue 730, and Corentin's comment on WebGPU issue 1522). I don't have a Mac device to try though. But the log should be reliable. And just as you said. Depth/stencil formats for 1D textures can be buggy in real world drivers.
Done.